evaluator/abstract_evaluator.py

```diff
@@ -73,11 +81,25 @@ def get_config_file_error_message(cls, error):
             f". (error: {error})"
 
     @classmethod
-    def get_evaluator_config(cls):
+    def get_specific_config(cls, raise_exception=True, raw_file=False):
+        try:
+            if raw_file:
+                with open(cls.get_config_file_name()) as file:
+                    return file.read()
```

Comment:
Where are raw files used ?

Reply:
they are required in this format to be parsed in web interface js: then passed to config editor

Reply:
Ok its the raw json directly parsed by the JS library